cadquery/occ_impl/solver.py

```diff
@@ -39,12 +38,10 @@
 
 NoneType = type(None)
 
-DOF6 = Tuple[float, float, float, float, float, float]
+DOF6 = Tuple[Tuple[float, float, float], Tuple[float, float, float]]
 ConstraintMarker = Union[gp_Pln, gp_Dir, gp_Pnt, gp_Lin, None]
 
-UnaryConstraintKind = Literal[
-    "Fixed", "FixedPoint", "FixedAxis", "FixedRotation", "FixedRotationAxis"
```

Comment:
It shouldn't be a problem as it wasn't part of a cadquery release.  It also wasn't documented and the purpose was more for internal use converting `FixedRotation` constraints into multiple `FixedRotationAxis` constraints.  Yes, a user *could* have manually added a `FixedRotationAxis` constraint which would now fail.

old
```assy.constrain("b1", "FixedRotationAxis", (2,45))```

convert to 
```assy.constrain("b2", "FixedRotation", (0,0,45))```

Reply:
In principle it could, but it was added in #975 so only recently and only to master. Moreover, it does not work and is an implementation detail for the previous version of `FixedRotation`.